GPflow/tf_wraps.py

```diff
@@ -20,13 +20,16 @@
 import os
 import tensorflow as tf
 from tensorflow.python.framework import ops
+import warnings
 from ._settings import settings
+import numpy as np
 
 
 def eye(N):
     """
     An identitiy matrix
     """
+    warnings.warn('tf_wraps.eye is deprecated: use tf.eye instead', np.VisibleDeprecationWarning)
```

Comment:
there's precedent for having deprecated function warnings in GPflow. We'll leave them in for a minor version and then remove them.